application-passwords.js

```diff
@@ -33,15 +33,23 @@
 				name : name
 			}
 		} ).done( function ( response ) {
+
 			$newAppPassField.prop( 'disabled', false ).val('');
 			$newAppPassButton.prop( 'disabled', false );
 
+			// Check if password was added
+
+			if ( typeof response.error !== 'undefined' ) {
+				alert(response.error);
+				return;
+			}
+
 			$newAppPassForm.after( tmplNewAppPass( {
 				name     : name,
 				password : response.password
 			} ) );
 
-			$appPassTbody.prepend( tmplAppPassRow( response.row ) );
+			$appPassTbody.append( tmplAppPassRow( response.row ) );
```

Comment:
Why switching to append instead of prepend?  I'd initially preferred prepend due to making the new one first-on-the-list for users to spot more easily.

If the generated list table is kicking in the other way, I'd sooner reverse it to keep more recent first if it's a ux issue.

Either way, probably better being a discrete pull request, as it's a ux decision, and not tied into displaying error messages.